### ISSUE TITLE
Bug in tutrorial code

### ISSUE THREAD

**Issue by Sum02dean** [vanvalenlab/deepcell-tf#398]
**TypeError: Invalid shape (512, 512, 1) for image data**
An issue with running final code block with deepcell-tf/notebooks/applications/Cytoplasm_Application.ipynb. 

**To Reproduce**
Run tutorial NB

**Expected behavior**
Error free plot of segmentation. 

**Screenshots**
![image](https://user-images.githubusercontent.com/36505689/82228991-22d0b200-992a-11ea-94e3-12bd31e6d4e4.png)

**Desktop (please complete the following information):**
 - OS: linux
 - Version deepcell 0.4
 - Python Version  3.6

**Solved**
Need to squeeze the input to remove a dimension. 
Change:

```
fig, ax = plt.subplots(1,2,figsize=(10,8))
ax[0].imshow(im[i,...,0])
ax[1].imshow(labeled_im[i,...,0](-1) , cmap='jet')
```

to

```
fig, ax = plt.subplots(1,2,figsize=(10,8))
ax[0].imshow(im[i,...,0])
ax[1].imshow(labeled_im[i,...,0].squeeze(-1) , cmap='jet')

```
** Result**
![image](https://user-images.githubusercontent.com/36505689/82229131-4f84c980-992a-11ea-82cf-0e0c1d2fc2a0.png)

**Comment by Sum02dean** [comment]
Provided a solution so marking as closed

**Comment by willgraf** [comment]
Thanks for this. I believe this update should be resolved in #397
